index.ts

```diff
@@ -20,16 +20,20 @@ const nameMaxLength = 80;
 // The longest description allowed, to avoid them growing into documentation.
 const descriptionMaxLength = 300;
 
+// The earliest possible Baseline low date, the release date of Edge 12.
+const earliestBaselineLowDate = '2015-07-29';
```

Comment:
I don't know if this is better exactly, but it feels more "correct":

```suggestion
// needs import { browser } from "compute-baseline/browser-compat-data";
const earliestBaselineLowDate = browser("edge").version("12").date.toString();
```

Reply:
OK, if this will eventually become a date, let's leave it as a date.